addon/components/session-overview.hbs

```diff
@@ -1,384 +1,371 @@
-          <span class="validation-error-message">{{v-get this "title" "message"}}</span>
+<div
+  data-test-session-overview
+  {{did-insert (perform this.load) @session @sessionTypes}}
+  {{did-update (perform this.load) @session @sessionTypes}}
+>
+  {{#unless this.load.isRunning}}
+    <div class="session-header">
+      <span class="title">
+        {{#if @editable}}
+          <EditableField
+            @value={{this.title}}
+            @save={{fn this.changeTitle}}
+            @close={{fn this.revertTitleChanges}}
+            @saveOnEnter={{true}}
+            @closeOnEscape={{true}} as |isSaving|
+          >
+            <Input
+              type="text"
+              @value={{this.title}}
+              @key-press={{fn this.addErrorDisplayFor "title"}}
+              disabled={{isSaving}}
+            />
+            {{#each (await (compute (fn this.getErrorsFor) "title")) as |message|}}
+              <span class="validation-error-message">
+                {{message}}
+              </span>
+            {{/each}}
+          </EditableField>
+        {{else}}
+          <h3>{{@session.title}}</h3>
         {{/if}}
-      {{/editable-field}}
-    {{else}}
-      <h3>{{@session.title}}</h3>
-    {{/if}}
-  </span>
-  <span class ="session-publication">
-    {{#if @editable}}
-
-      {{publish-menu
-        title=this.menuTitle
-        icon=this.menuIcon
-        publicationStatus=this.publicationStatus
-        showAsIs=this.showAsIs
-        showPublish=this.showPublish
-        showReview=this.showReview
-        showTbd=this.showTbd
-        showUnPublish=this.showUnPublish
-        reviewRoute="session.publication_check"
-        reviewObject=@session
-        parentObject=(await @session.course)
-        publishTranslation="general.publishSession"
-        unPublishTranslation="general.unPublishSession"
-        publish=(action "publish")
-        publishAsTbd=(action "publishAsTbd")
-        unpublish=(action "unpublish")
-      }}
-    {{else}}
-      {{publication-status item=@session}}
-    {{/if}}
-  </span>
-</div>
-
-<section class="session-overview">
-  <div class="last-update">
-    {{fa-icon icon="history" title=(t "general.lastUpdate")}}
-    {{t "general.lastUpdate"}}:
-    {{this.updatedAt}}
-  </div>
-
-  <div class="session-overview-header">
-
-    <div class="title">
-      {{t "general.overview"}}
-    </div>
-
-    <div class="session-overview-actions">
-      {{#if (and (is-fulfilled this.showCopy) (await this.showCopy))}}
-        {{#link-to "session.copy" @session class="copy"}}
-          {{fa-icon icon="copy" title=(t "general.copySession") fixedWidth=true}}
-        {{/link-to}}
-      {{/if}}
-    </div>
-  </div>
-
-  <div class="session-overview-content">
-    <div class="sessiontype block">
-      <label>{{t "general.sessionType"}}:</label>
-      <span>
+      </span>
+      <span class="course-publication">
```

Comment:
Copy paste? Doesn't do anything, maybe should do something like [this](https://github.com/ilios/common/blob/master/app/styles/ilios-common/components/course-header.scss#L38)?

Reply:
c/p indeed. good catch! fixed.